opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -541,7 +542,10 @@ CpGrid::scatterGrid(EdgeWeightMethod method,
     void CpGrid::processEclipseFormat(const grdecl& input_data, double z_tolerance,
                                       bool remove_ij_boundary, bool turn_normals)
     {
-        current_view_data_->processEclipseFormat(input_data, {}, z_tolerance, remove_ij_boundary, turn_normals);
+        using NNCMap = std::set<std::pair<int, int>>;
+        using NNCMaps = std::array<NNCMap, 2>;
+        NNCMaps nnc;
+        current_view_data_->processEclipseFormat(input_data, nullptr, nullptr, nnc, z_tolerance, remove_ij_boundary, turn_normals);
```

Comment:
Not totally sure that I understand your question correctly here. 

For this situation, nnc will always be empty. The only reason it can be modified within this function call is numerical aquifers, but we pass in `nullptr` for `EclipseState*`, so no numerical aquifers.  `nnc` created here is purely for interface.  It does not look pretty.  

Suggestion is very welcomed here.